src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/mappers/DocumentMapper.java

```diff
@@ -14,19 +12,21 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public interface ModelMapper<T extends CaseData> {
+public class DocumentMapper {
```

Comment:
utility classes are final classes

Reply:
Just did a quick search on `// util class` and existing helper classes in this repo are not final.
As pmd and/or sonar don't complain, I'll leave it as is and spare myself 20min ci build waiting time ;)

Reply:
ᕕ( ᐛ )ᕗ